corporate/lib/stripe.py

```diff
@@ -99,6 +99,19 @@ def format_money(cents: float) -> str:
     return f"{dollars:.{precision}f}"
 
 
+def format_discount_percentage(discount: Optional[Decimal]) -> Optional[str]:
```

Comment:
<img width="816" alt="Screenshot 2023-11-23 at 10 33 29 PM" src="https://github.com/zulip/zulip/assets/25124304/1533a407-b967-4924-81e7-4da503882f84">

<img width="816" alt="Screenshot 2023-11-23 at 10 15 05 PM" src="https://github.com/zulip/zulip/assets/25124304/de4f8a18-db2f-48aa-be37-50f15b67a9b2">

There was not enough space on the right of billing page charge calculation. So, I placed the discount on the next line.